Traceback (most recent call last):
  File "lib/cc.py", line 84, in <module>
    results=cc.evaluate(file)
  File "lib/cc.py", line 19, in evaluate
    red = redbaron.RedBaron(file.read())
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/redbaron/redbaron.py", line 36, in __init__
    self.node_list = base_nodes.NodeList.from_fst(baron.parse(source_code), parent=self, on_attribute="root")
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/baron/baron.py", line 49, in parse
    tokens = tokenize(source_code, False)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/baron/baron.py", line 70, in tokenize
    return mark_indentation(inner_group(space_group(_tokenize(group(split(pouet)), print_function))))
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/baron/indentation_marker.py", line 24, in mark_indentation
    return list(mark_indentation_generator(sequence))
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/baron/indentation_marker.py", line 85, in mark_indentation_generator
    the_indentation_level_changed = get_space(current) is None or get_space(current) != indentations[-1]
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/baron/indentation_marker.py", line 38, in get_space
    if len(node) < 3 or len(node[3]) == 0:
IndexError: tuple index out of range
